src/stdx.zig

```diff
@@ -202,3 +202,33 @@ pub const log = if (builtin.is_test)
     }
 else
     std.log;
+
+/// Inline version of Google Abseil "LowLevelHash" (inspired by wyhash).
+/// https://github.com/abseil/abseil-cpp/blob/master/absl/hash/internal/low_level_hash.cc
+pub inline fn hash_inline(value: anytype) u64 {
```

Comment:
I think there might be a bit of a footgun opportunity here with `value: anytype` - eg, `hash_inline("hello world")` won't do what you expect.

Reply:
Other than that - perhaps some hardcoded tests to compare to Abseil output?